src/Uno.UI/UI/Xaml/ResourceDictionary.cs

```diff
@@ -126,5 +228,29 @@ public void CopyTo(global::System.Collections.Generic.KeyValuePair<object, objec
 		public global::System.Collections.Generic.IEnumerator<global::System.Collections.Generic.KeyValuePair<object, object>> GetEnumerator() => _values.GetEnumerator();
 
 		global::System.Collections.IEnumerator global::System.Collections.IEnumerable.GetEnumerator() => _values.GetEnumerator();
+
+		private static class Themes
+		{
+			public const string Default = "Default";
+			public static string Active
+			{
+				get
+				{
+					if (Application.Current == null)
+					{
+						return "Light";
+					}
+					switch (Application.Current.RequestedTheme)
+					{
+						case ApplicationTheme.Light:
+							return "Light";
+						case ApplicationTheme.Dark:
+							return "Dark";
+						default:
+							throw new InvalidOperationException($"Theme {Application.Current.RequestedTheme} is not valid");
```

Comment:
That's not correct: we support other themes like "HighContrast", etc...  even the "Pink" custom theme on the calculator.

Reply:
`HighContrast` is a good point, I didn't do any testing for that.

Do you recommend any changes re: the custom theme feature? Note that this particular code is correct because the only valid values of the `ApplicationTheme` enum are `Light` and `Dark`, but we can use the custom theme in the lookup. Does it replace the Light/Dark theme? Complement it? Are there docs somewhere?